pkg/varlinkapi/pods.go

```diff
@@ -247,7 +247,7 @@ func (i *LibpodAPI) RemovePod(call iopodman.VarlinkCall, name string, force bool
 	if err != nil {
 		return call.ReplyPodNotFound(name, err.Error())
 	}
-	if err = i.Runtime.RemovePod(ctx, pod, force, force); err != nil {
+	if err = i.Runtime.RemovePod(ctx, pod, true, force); err != nil {
```

Comment:
This fixes the issue for podman remote `pod prune` remove stop containers.
But  podman remote `pod rm` will use the same logic to remove pod and stopped containers from the pod.
Do we allow `podman pod rm` to remove pod if containers are stopped?
@rhatdan @mheon

Reply:
We should, that is the change we made to podman pod rm.

Reply:
From the tests failures  `podman pod rm `doesn't remove a pod with a container https://github.com/containers/libpod/blob/b3d8b99b82d466ab9a325e66c1a240ff989eeb66/test/e2e/pod_rm_test.go#L80-L89

Reply:
Then remove this test.  Podman pod will always have a container.  I don't see a reason to not remove the pod.  Too much work on the user to have to remove all containers before removing the pod.  Force says remove the pod and containers if they are running.
Containers in Pods do not standalone, so pod includes containers.

Reply:
We handle the infra container explicitly. I would prefer that we keep the current behavior - `--force` to remove pods with anything other than an infra container

Reply:
Why.  Containers are part of the Pod and should be treated as such. Similar to built in volumes.  I don't want to add yet another option that will confuse the users.

When people think in terms of pods, they do not consider the containers as separate.

Reply:
And `podman rm` doesn't remove volumes unless you tell it to - you need `--volumes` to do that.

Reply:
Honestly, we should just get a session together about pods at the F2F - it's a good opportunity to revisit these decisions

Reply:
podman rm removes the builtin volumes, IE Volumes that are not used by other entities, I see Pod containers the same way.  But yes we can talk about this at the Face2Face.

@haircommander @mrunalp What does kubernetes do.  Does it first remove all of the containers from a pod, before removing the pod, or when you tell CRI-O to remove the pod, then it removes all of the containers and finally the POD?

Reply:
if the kubelet asks cri-o to remove a pod, it is removed, regardless of containers that live in it, I am pretty sure. The kubelet making a request of cri-o is a bit different than a user typing cli, though. I think ultimately I'm with @rhatdan here, a pod should be treated as a unit.